main.py

```diff
@@ -387,6 +383,7 @@ def percent(part, whole):
 		summary['total_bugs'] = "Blocker Bugs: {} total, {} unique".format(len(all_bugs), len(unique_bugs))
 
 	# ticket metrics
+	all_tickets = [ticket_id for ticket_id in all_tickets if ticket_id != 0]
```

Comment:
Loop is overkiil IMO to get rid of a known key:value pair.
See: https://thispointer.com/different-ways-to-remove-a-key-from-dictionary-in-python/
Doesn't require iterators

Reply:
Beautiful; I will incorporate this for Bugzilla as well in this patch

Reply:
So looking back at the code, this (`all_bugs`/`all_tickets`) is actually a list and not a dictionary, so I think we may need to keep them as list comprehensions

Reply:
You, sir, are absolutely right. My mistake. So the remove command should work knowing that there's only a single zero value (because we make a set into the all_tickets right?:
https://stackoverflow.com/questions/2793324/is-there-a-simple-way-to-delete-a-list-element-by-value

Reply:
There are multiple 0's, because `all_tickets` is added to for every val including duplicates.

Reply:
@nathan-weinberg And does it make sense to create a set? or do you need it for the count...
I think it's the latter. In which case I can't see another solution except for yours in this short term.
Once we modularize, we can keep counters differently...

Reply:
We just need it for the count, the set still significantly cuts down on API calls

Agreed on modualizing